src/Livewire/Servers.php

```diff
@@ -59,4 +71,18 @@ public function placeholder(): Renderable
     {
         return View::make('pulse::components.servers-placeholder', ['cols' => $this->cols, 'rows' => $this->rows, 'class' => $this->class]);
     }
+
+    /**
+     * Determine if the system should be ignored.
+     *
+     * @param  object{ timestamp: int, key: string, value: string }  $system
+     */
+    protected function ignoreSystem(object $system): bool
+    {
+        if ($this->ignoreAfter === null) {
+            return false;
+        }
+
+        return CarbonImmutable::createFromTimestamp($system->timestamp)->addSeconds($this->secondsUntil($this->ignoreAfter))->isPast();
+    }
```

Comment:
I've called this `ignoreSystem` to differentiate it from our shared `ignore` method on the `Ignores` trait.